README.md

```diff
@@ -53,6 +53,36 @@ Or install it yourself as:
     message = MyMessage.new({:first_property => 1, :another_property => "another"})
     message.publish
 
+    # create a message class including changesets
+    class MyChangesetMessage
+      include Emque::Producing.message(:with_changeset => true)
+      topic "topic1"
+      message_type "mymessage.new"
+
+      # Need to override an attribute name in the changeset? Simply define
+      # the old name and new name here. This can be useful for ensuring
+      # messages are consistent across varying producers.
+      alias_changeset_attrs :old_attr_name => :new_attr_name
```

Comment:
The only thing I'd say is traditionally you only alias one attribute per call, instead of multiple. So

```ruby
alias_changeset_attr :old_attr_name1, :new_attr_name1
alias_changeset_attr :old_attr_name2, :new_attr_name2
```

More traditional is all.